prometheus-redis_exporter.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: prometheus-redis_exporter
```

Comment:
Should we name this -exporter? I don't think apk cares, but just for consistency

Reply:
Addressed